### ISSUE TITLE
squib error on Ubuntu 22.04  "undefined symbol: g_bookmark_file_get_type"

### ISSUE THREAD

**Issue by cdesch** [andymeneely/squib#382]
**Describe the bug**
I receive the following error when running `squib new my_project` or any `squib ...` command on Ubuntu 22.04
```
ruby: symbol lookup error: /home/my_home/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/glib2-4.2.0/lib/glib2.so: undefined symbol: g_bookmark_file_get_type
```

**To Reproduce**

```
❯ squib new my_project
ruby: symbol lookup error: /home/my_home/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/glib2-4.2.0/lib/glib2.so: undefined symbol: g_bookmark_file_get_type
```

**Expected behavior**
For it to work....: -) 

**Environment**
 - OS: 
```
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy
```
 - Ruby version: 3.2.2

**Additional context**

**Check the following:**

- [X] I have updated Squib to the latest version.
- [X] I have checked the Squib issues backlog for something similar
- [X] I have attempted to make a minimal script demonstrating my problem
- [X] I have reviewed the [Squib Docs](https://squib.readthedocs.io/) and there's still a problem

**Comment by andymeneely** [comment]
Interesting. I don't have an Ubuntu machine these days, but the glib2 sometimes needs a separate `apt install` that wasn't called on installation. Did you get any errors on installation?

**Comment by cdesch** [comment]
No. It seems to install fine with `sudo apt-get install libglib2.0-dev`. I've been able to work with squib using docker for the interim:


Create new squib project:

```sh
docker run --rm -v "$PWD":/usr/src/app andymeneely/squib squib new my_card_game
```

Generate Cards

```sh
docker run --rm -v "$PWD":/usr/src/app andymeneely/squib ruby deck.rb
```

**Comment by cmembreno2** [comment]
Was this issue solved? and how? Facing the same in Linux Ubuntu,

ruby: symbol lookup error: /home/membrish1989/.rvm/gems/ruby-3.0.6/gems/glib2-4.2.2/lib/glib2.so: undefined symbol: g_bookmark_file_get_type
